platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/StsRealm.java

```diff
@@ -122,64 +134,63 @@ public void setContextPolicyManager(ContextPolicyManager contextPolicyManager) {
   /** Determine if the supplied token is supported by this realm. */
   @Override
   public boolean supports(AuthenticationToken token) {
-    boolean supported =
-        token != null && token.getCredentials() != null && token instanceof STSAuthenticationToken;
-
-    if (supported) {
-      LOGGER.debug("Token {} is supported by {}.", token.getClass(), StsRealm.class.getName());
-    } else if (token != null) {
-      LOGGER.debug("Token {} is not supported by {}.", token.getClass(), StsRealm.class.getName());
-    } else {
+    if (token == null) {
```

Comment:
📘 The `StsRealm` can handle the authentication token as long as it has a saml assertion on it.